guides/source/active_record_validations.md

```diff
@@ -713,20 +778,23 @@ class Order < ApplicationRecord
 end
 ```
 
-NOTE: If you want to ensure that the association is both present and valid,
-you also need to use `validates_associated`. More on that
-[below](#validates-associated)
+NOTE: If you want to ensure that the association is both present and valid, you
+also need to use `validates_associated`. More on that in the
+[validates_associated section](#validates-associated).
 
-If you validate the absence of an object associated via a `has_one` or
-`has_many` relationship, it will check that the object is neither `present?` nor
-`marked_for_destruction?`.
+If you validate the absence of an object associated via a
+[`has_one`](association_basics.html#the-has-one-association) or
+[`has_many`](association_basics.html#the-has-many-association) relationship, it
+will check that the object is neither `present?` nor `marked_for_destruction?`.
 
-Since `false.present?` is false, if you want to validate the absence of a boolean
-field you should use `validates :field_name, exclusion: { in: [true, false] }`.
+Since `false.present?` is false, if you want to validate the absence of a
+boolean field you should use `validates :field_name, exclusion: { in: [true,
+false] }`.
```

Comment:
I think the example here is a bit more readable if we use the triple backticks:
````suggestion
Since `false.present?` is false, if you want to validate the absence of a
boolean field you should use:

```ruby
validates :field_name, exclusion: { in: [true, false] }
```
````